converter/converter_test.go

```diff
@@ -579,8 +578,8 @@ func TestToPURL(t *testing.T) {
 				Locations: []string{"irrelevant"},
 				Version:   "irrelevant",
 			},
-			wantErr: true,
-			onGoos:  "linux",
+			want:   nil,
```

Comment:
Change test description to "returns nil on Linux"

Reply:
Thanks, done!

I noticed that `TestToCPEs` never actually added an error test case.  Do you want me to add a nil-response test case?

Reply:
Good point - Yeah I think it'd be good to have that in there for consistency

Reply:
Added this to the merge commit - https://github.com/google/osv-scalibr/commit/826c2e6d65a6e066852c0ecc3c0a440b86bbfb76